src/actions/public/getBlock.ts

```diff
@@ -92,6 +93,18 @@ export async function getBlock<
   account extends Account | undefined,
   includeTransactions extends boolean = false,
   blockTag extends BlockTag = 'latest',
+>(
+  client: Client<Transport, chain, account>,
+  params: GetBlockParameters<includeTransactions, blockTag> = {},
+): Promise<GetBlockReturnType<chain, includeTransactions, blockTag>> {
+  return getAction(client, _getBlock, 'getBlock')(params)
+}
```

Comment:
```suggestion
>(
  client: Client<Transport, chain, account>,
  params: GetBlockParameters<includeTransactions, blockTag> = {},
): Promise<GetBlockReturnType<chain, includeTransactions, blockTag>> {
  return getAction(client, _getBlock, 'getBlock')(params)
}
```

Might be able to do this just to save some boilerplate

```typescript
const withGetAction = <TAction>(action: TAction, name: string): TAction => {
  return ((client, params) => {
     return getAction(client, action, name)(params)
  }) as unknown as TAction
}

export const getBlock = withGetAction(client, action, 'getBlock')
```

Reply:
DOn't do this it's convenient but will make stack traces suck